internals/secrethub/tpl/v2_test.go

```diff
@@ -1,23 +1,496 @@
-package tpl_test
+package tpl
 
 import (
+	"bytes"
 	"testing"
 
-	"github.com/secrethub/secrethub-cli/internals/secrethub/tpl"
-	generictpl "github.com/secrethub/secrethub-cli/internals/tpl"
+	"github.com/secrethub/secrethub-cli/internals/secrethub/tpl/fakes"
+
 	"github.com/secrethub/secrethub-go/internals/assert"
 )
 
+func TestParserV2_parse(t *testing.T) {
+	cases := map[string]struct {
```

Comment:
Am I correct that the following case is missing? `{{ foo/bar }a` 

I couldn't get it to output a correct error. It gave "illegal character in secret ' '`. Or does that make sense?

Reply:
After discussion we've chosen to rephrase the error to `unexpected character 'a', expected '}'`